src/ActionList/ActionList.features.stories.tsx

```diff
@@ -41,6 +42,109 @@ export const SimpleList = () => (
   </ActionList>
 )
 
+export const WithHeading = () => (
```

Comment:
I think we should add this heading to docs and all stories in another PR after this, to show that this is the recommended usage.

Reply:
Oh that is right! All should have a heading - Not sure what I was thinking when I added a story with the heading! 😆  I can do them in this PR as well - to keep everything contained in one PR because they wouldn't be complex to review anyway.